File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int Height;
    // Ask for the pyramid's height until the value = [1,8]
    do
    {
        Height = get_int("Height: ");
    }
    while (Height < 1 || Height > 8);
    // Print the pyramid
    for(int i = 1; i <= Height; i++)
    {
        for(int j = Height; j >= i + 1; j--)
        {
            printf(" ");
        }
        for (int k = 1; k <= i; k++)
        {
            printf("#");
        }
        printf("  ");
        for (int k = 1; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: mario.py
#!/usr/bin/env python3

from cs50 import get_int


def block():
    print("#", end="")


def air():
    print(" ", end="")


def pyramid(height):
    for i in range(1, height + 1, 1):
        for j in range(height + 1, i + 1, -1):
            air()
        for x in range(i):
            block()
        print("")


def main():
    while True:
        height = get_int("Height: ")
        if height > 0 and height < 9:
            break
    pyramid(height)


if __name__ == "__main__":
    main()